### DISCUSSION TITLE
Do we need an open source guide for open source projects?

### DISCUSSION THREAD

**Discussion by CsatariGergely** [todogroup/ospology#110] [OSPO Discussions]
Hi, 
From time to time I ran into problems that an open source project what I would like to use is difficult to consume due to some licensing/legal issues or my contribution is not welcomed for the same reason. I mean situations, like
- Declared license of the project is not clear
- There are no copyright notices
- The project provides a container, but it is not clear how the container is build or what does it contain
- Project commiters freak out about copyright notices
- Charter of the project states that they license documentation using CC license, but still they have only an Apache License in their repo (and they refuse to change it)

I think some porjects would really need a short guide about the legal framework of open source and an explanation what is important for the ones consuming their projects and the ones who would like to contribute. 

What to you think?

**Comment by craigez** [comment]
Or should their be a guide on how to work with these communities/maintainers to resolve the ambiguity?

**Reply by CsatariGergely** [reply]
Maybe both of these are needed :)

**Comment by mwichmann** [comment]
There are really a bunch of these from the viewpoint of setting up a welcoming, easy to work with community.  Since GitHub hosts so many of these, they've curated some guides on the topic:

https://opensource.guide/building-community/

The legal part is often mentioned (pick a good license, make it prominent, make sure tools are picking up that license).

That doesn't mean there still aren't many projects that need help, and many people that need help to respectfully interact with projects that are more .... difficult ... in their setup.    The container example is actually a good one - you can have a fairly clear project, then to make it easier for people to consume you package it up into _something_ - docker, snap, flatpak, whatever, and somehow the picture is less clear now.
